petastorm/py_dict_reader_worker.py

```diff
@@ -226,11 +226,13 @@ def _load_rows_with_predicate(self, pq_file, piece, worker_predicate, shuffle_ro
             return filtered_decoded_predicate_rows
 
     def _read_with_shuffle_row_drop(self, piece, pq_file, column_names, shuffle_row_drop_partition):
+        # If integer_object_nulls is set to False, nullable integer fields are return as floats
+        # with nulls translated to nans
```

Comment:
Did you also update the doc of `to_pandas` ?

Reply:
`to_pandas` is part of pyarrow.

Reply:
`to_pandas` is part of pyarrow.